### PR TITLE
Merge refactor website

### PR DESCRIPTION
**Pull request by AlbumenJ**

### CODE REVIEW

File: layouts/_default/search.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
+<!doctype html>
```
Comment (AlbumenJ) [inline layouts/_default/search.html:1]: Should we remove this html? @chickenlj

File: layouts/case-studies/list.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
+{{ define "main" }}
```
Comment (AlbumenJ) [inline layouts/case-studies/list.html:1]: also for this

File: layouts/partials/blog-meta-links.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div class="td-page-blog-meta ml-2 pb-1 pt-2 mb-4">
```
Comment (AlbumenJ) [inline layouts/partials/blog-meta-links.html:1]: also

File: layouts/partials/feedback.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
+{{ if not .Params.hide_feedback }}
```
Comment (AlbumenJ) [inline layouts/partials/feedback.html:1]: also

File: layouts/partials/handle-version-switch-404.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{{/*
```
Comment (AlbumenJ) [inline layouts/partials/handle-version-switch-404.html:1]: also

File: layouts/shortcodes/blocks/dubbo-features.html [hunk 6]
```diff
@@ -0,0 +1,36 @@
+{{/*
```
Comment (AlbumenJ) [inline layouts/shortcodes/blocks/dubbo-features.html:1]: also

**Review by chickenlj — APPROVED** [review]
LGTM.

**Pull request merged**